app/models/restaurant.rb

```diff
@@ -1,7 +1,12 @@
+
```

Comment:
empty lines everywhere
????

we normally have annotations here. its a cool gem that adds the columns / schema into the model file (gem named annotate)

Reply:
done.